packages/config/config/devices/0x044c/myot2.json

```diff
@@ -0,0 +1,201 @@
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Boiler Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum Domestic Hot Water Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Domestic Hot Water Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "5",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum Cool Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "6",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Cool Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "10",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Flag",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "11",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Code",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "12",
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Current Error Sub Code",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "20",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Pump State",
+			"readOnly": true
+		},
+		{
+			"#": "21",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Compressor State",
+			"readOnly": true
+		},
+		{
+			"#": "22",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Operation Mode",
+			"readOnly": true
+		},
+		{
+			"#": "31",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Summer Mode"
+		},
+		{
+			"#": "30",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Domestic Hot Water Enable"
+		},
+		{
+			"#": "50",
+			"$import": "~/templates/master_template.json#base_0-65535",
+			"label": "Load Power",
+			"unit": "0.01 W",
+			"readOnly": true
+		},
+		{
+			"#": "51",
+			"$import": "~/templates/master_template.json#base_0-65535",
+			"label": "Battery Voltage Output",
+			"unit": "0.01 V",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "52",
+			"$import": "~/templates/master_template.json#base_0-65535",
+			"label": "Battery Current Output",
+			"unit": "0.01 A",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "53",
+			"$import": "~/templates/master_template.json#base_0-65535",
+			"label": "Battery Power Output",
+			"unit": "0.01 W",
+			"readOnly": true
+		},
```

Comment:
Is it on purpose that some of these params set valueSize to 2, and some leave it on 4 (from the $import)?

Reply:
Yes, accordingly to the unofficial document received from the manufacturer.
[MyOT v3.pdf](https://github.com/zwave-js/node-zwave-js/files/9021342/MyOT.v3.pdf)
.